### PR TITLE
ClientApp saves JWTtoken

### PR DESCRIPTION
**Pull request by ArinaSavchenko**

### CODE REVIEW

File: ClientApp/src/app/Services/user.service.ts [hunk 1]
```diff
@@ -22,11 +24,27 @@ export class UserService {
...
+
+  get isLoggedIn(): boolean {
+    return localStorage.getItem('token') !== null;
+  }
+
+  logout(): void {
+    localStorage.removeItem('token');
+  }
+
+  errorHandling(error: HttpErrorResponse): Observable<any>{
```
Comment (andrei-kruhliakou-itechart) [inline ClientApp/src/app/Services/user.service.ts]: Name of any method should start with a verb. Something like 'handleError' will be better as a name.
Also missing space before {

File: ClientApp/src/app/log-in/log-in.component.html [hunk 2]
```diff
@@ -1,32 +1,39 @@
...
+  <mat-dialog-actions>
+    <button mat-raised-button class="logInButton" [disabled]="!userInfo.valid" (click)="logIn()">Log in</button>
+  </mat-dialog-actions>
 
-  <p *ngIf="message" id="error_message" class="mat-error">{{message}}</p>
-<p class="message">No account yet?</p>
-<a routerLink="/airline/registration" class="singUpLink" (click)="closeDialog()">Sing up</a>
+    <p *ngIf="message" id="error_message" class="mat-error">{{this.message}}</p>
+  <p class="message">No account yet?</p>
+  <a routerLink="/airline/registration" class="singUpLink" (click)="closeDialog()">Sing up</a>
```
Comment (andrei-kruhliakou-itechart) [inline ClientApp/src/app/log-in/log-in.component.html]: Typo 'Sing'

File: ClientApp/src/app/log-in/log-in.component.ts [hunk 3]
```diff
@@ -19,14 +21,24 @@ export class LogInComponent {
 
  constructor(
    public dialogRef: MatDialogRef<LogInComponent>,
-   private userService: UserService) {}
+   public userService: UserService) {}
```
Comment (andrei-kruhliakou-itechart) [inline ClientApp/src/app/log-in/log-in.component.ts:24]: Why did you change this to public?

File: ClientApp/src/app/registration/registration.component.ts [hunk 4]
```diff
@@ -4,6 +4,8 @@ import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms'
 import { RegisterModel } from '../Models/RegisterModel';
 import { UserService } from '../Services/user.service';
 import { PasswordsMatchValidator } from '../Validators/PasswordsMatchValidator';
+import { ResponseModel } from '../Models/ResponseModel';
+import { Router, ActivatedRoute } from '@angular/router';
```
Comment (andrei-kruhliakou-itechart) [inline ClientApp/src/app/registration/registration.component.ts]: Please move this to angular imports

**Review by andrei-kruhliakou-itechart — CHANGES_REQUESTED** [review]

File: ClientApp/src/app/log-in/log-in.component.ts (hunk 3 above)
Comment (ArinaSavchenko) [reply ClientApp/src/app/log-in/log-in.component.ts:24]: I use this service in HTML file for this component to check if user is logged in. If they are private I don't have access to them

**Review by andrei-kruhliakou-itechart — APPROVED** [review]
